### ISSUE TITLE
Decompress multiple images

### ISSUE THREAD

**Issue by YannDubs** [InterDigitalInc/CompressAI#13]
Hi, 

Right now you cannot decompress multiple images using `EntropyBottleneck.decompress` (but you can compress multiple images at once). As a result `bttleneck.decompress(bttleneck.compress(images), shape)` would brake. This is because on this [line](https://github.com/InterDigitalInc/CompressAI/blob/master/compressai/entropy_models/entropy_models.py#L462) you use `self._medians().detach().view(1, -1, 1, 1)` (i.e. batch size is one for medians even though it is `batch_size` for indices) but on [this line](https://github.com/InterDigitalInc/CompressAI/blob/master/compressai/entropy_models/entropy_models.py#L251) you actually explicitly ensure that the batch size for indices and medians is the same. 

During compression you don't have this problem because you don't explicitly test for equal batch_size and broadcasting takes care of the rest. I would suggest one of the following simple solutions
-  either removing the explicit ensuring of equal batch size (might brake downstream though)
- simply use `self._medians().detach().expand(len(strings), -1, 1, 1)` instead of `self._medians().detach().view(1, -1, 1, 1)`  [here](https://github.com/InterDigitalInc/CompressAI/blob/master/compressai/entropy_models/entropy_models.py#L462). This only uses an expanded view so there's no memory usage increase.

**Comment by jbegaint** [comment]
Hi Yann :) thanks for the bug report! Good point, it's a valid use case we haven't correctly tested!
I'll work on this.
Thanks!

**Comment by jbegaint** [comment]
Yann, this should be fixed via ab6a7af51d10f344026e68f89f3f91d2f0c9da56. Let me know how it goes :-).

**Comment by YannDubs** [comment]
works as desired, thanks :)
